terraform/aws/analytical-platform-development/control-panel-message-broker/sqs.tf

```diff
@@ -0,0 +1,20 @@
+resource "aws_sqs_queue" "s3_queue" {
+  #ts:skip=AWS.SQS.NetworkSecurity.High.0570
+  name = "s3_queue"
+
+  tags = var.tags
+}
+
+resource "aws_sqs_queue" "iam_queue" {
+  #ts:skip=AWS.SQS.NetworkSecurity.High.0570
+  name = "iam_queue"
+
+  tags = var.tags
```

Comment:
```suggestion
  tags = var.tags
```

Reply:
this isn't required anymore, we set default tags with the provider 😎